### DISCUSSION TITLE
The xbox app doesnt let me sign in now and I cant download my game bc of it

### DISCUSSION THREAD

**Discussion by alt-n5683** [builtbybel/ThisIsWin11#116] [General]
The xbox app doesnt let me sign in now and I cant download my game bc of it.
I did the gaming profile and uninstalled the xbox apps that came with that were broke and then reinstalled and the open and work just wont sign in.

**Comment by Belim** [comment]
Check here https://github.com/builtbybel/ThisIsWin11/issues/53
